go/client/identify_ui.go

```diff
@@ -23,6 +23,30 @@ func NewIdentifyTrackUIProtocol(g *libkb.GlobalContext) rpc.Protocol {
 	return keybase1.IdentifyUiProtocol(&IdentifyUIServer{ui})
 }
 
+func cliIsLoggedIn(g *libkb.GlobalContext) bool {
+	configCli, err := GetConfigClient(g)
+	if err != nil {
+		return false
+	}
+
+	currentStatus, err := configCli.GetCurrentStatus(context.TODO(), 0)
+	if err != nil {
+		return false
+	}
+
+	return currentStatus.LoggedIn
+}
+
+// NewIdentifyMaybeTrackUIProtocol returns either IdentifyUiProtocol with
+// IdentifyTrackUI or IdentifyUI depending if user is logged in or not.
+func NewIdentifyMaybeTrackUIProtocol(g *libkb.GlobalContext) rpc.Protocol {
+	if !cliIsLoggedIn(g) {
```

Comment:
hm, can I use the `Context` that is passed back from the service to e.g. `func (i *IdentifyUIServer) Confirm` to see if user is logged in? or it isn't the right Context type?